engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/content-host-packages-applicable.controller.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostPackagesApplicableController
+ *
+ * @requires $scope
+ * @resource $timeout
+ * @resource $window
+ * @requires HostPackage
+ * @requires translate
+ * @requires Nutupane
+ * @requires BastionConfig
```

Comment:
I think you're missing `@requires Package` here. Also, BastionConfig isn't used.